packages/core/src/modules/member/member.worker.ts

```diff
@@ -61,6 +62,12 @@ export const memberWorker = registerWorker(
       .with({ name: 'student.statuses.new' }, ({ data }) => {
         return createNewActiveStatuses(data);
       })
+      .with(
+        { name: 'student.company.review.notifications.weekly' },
```

Comment:
let's change the name to be `student.company_review_notifications`. Make sure to also update in bull.types.ts